### PR TITLE
add dino

### PR DESCRIPTION
**Pull request by etendra2501**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2264,3 +2264,8 @@ harshithap2000-dai_deprivation"
 "sulamanshahbukhari_dino"
 
 ![](sulamanshahbukhari_dino.png)
+
+"etendra2501_dino_dino"
+
+![](https://github.com/etendra2501/dinosaurs/blob/etendra2501_dino_dino/etendra2501_dino_dino.png)
```
Comment (exu3) [inline README.md:2270]: Could you upload your dino as a file to the repo instead adding it as a link?

Comment (etendra2501) [reply README.md]: done

**Review by priyankarpal — CHANGES_REQUESTED** [review]
There is a conflict please resolve it.

**Review by sampoder — APPROVED** [review]
👓

**Pull request merged**